tests/unit/tests/PaymentGateways/Stripe/ApplicationFeeTest.php

```diff
@@ -91,4 +91,18 @@ public function testHasLicenseAllAccessPass() {
             $this->gate->hasLicense( 'give-stripe' )
         );
     }
+
+    public function testIsStripeAccountFromBrazil() {
```

Comment:
Let's rephrase this as a test of `isCountrySupportApplicationFee` as opposed to the specific instance of Brazil.